.pre-commit-config.yaml

```diff
@@ -1,12 +1,45 @@
+# https://pre-commit.com/
```

Comment:
suggestion: please move the file into another PR as it is not related to the certificate problem tackled here.

The linters are really useful and tackled in #580.